src/cljc/athens/common_db.cljc

```diff
@@ -245,6 +245,32 @@
             x)))
 
 
+(defn between
+  "http://blog.jenkster.com/2013/11/clojure-less-than-greater-than-tip.html"
+  [s t x]
+  (if (< s t)
```

Comment:
More concise way:
```clojure
(< (min s t) x (max s t))
```

Reply:
Yeah for this I just cut pasted from the previous code, should have self-reviewed this first.